### PR TITLE
Add Ruby 2.3 to Travis build

### PR DESCRIPTION
**Pull request by greysteil**
That.

### CODE REVIEW

**Comment by greysteil** [comment]
Didn't know that, but makes perfect sense. I've added them back!

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,17 @@
 language: ruby
+
 rvm:
+  - "2.3.0"
   - "2.2"
   - "2.1"
   - "2.0"
   - "jruby-9.0.0.0"
+
+sudo: false
```
Comment (greysteil) [inline .travis.yml]: This moves us over to Travis's newer container-based infrastructure.

File: circle.yml [hunk 2]
```diff
@@ -1,3 +0,0 @@
-machine:
```
Comment (greysteil) [inline circle.yml]: Don't think we have any need for Circle in addition to Travis.

File: .travis.yml [hunk 3]
```diff
@@ -1,11 +1,17 @@
 language: ruby
+
 rvm:
+  - "2.3.0"
```
Comment (greysteil) [inline .travis.yml]: Travis doesn't have a `2.3` binary yet, so we need to specify the full version here.

**Comment by greysteil** [comment]
@michaelklishin - are you :+1: ?

**Comment by michaelklishin** [comment]
@greysteil yes, it looks good. I am not sure why we also have Circle CI, by the way.

**Comment by michaelklishin** [comment]
Thank you!

**Pull request merged**